app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,68 @@
+module Api
+  module V1
+    class CommentsController < BaseController
+      before_action :set_story
+
+      def index
+        comments = @story.comments.includes(:owner).all
+        render json: comments
+      end
+
+      def show
+        comment = @story.comments.find(params[:id])
+        render json: comment
+      end
+
+      def create
+        comment = Comment.new(
+          commentable: @story,
+          owner: current_user,
+          content: comment_params[:content]
+        )
+
+        if comment.save
```

Comment:
it authorizes you have access to the project so I think if you can read the project you’re allowed to comment